src/metadata_validator.py

```diff
@@ -44,17 +48,18 @@ def metadataValidate(configs, job_queue, mongo_dao):
                     if data.get(SQS_TYPE) == "Validate Metadata" and data.get(SUBMISSION_ID) and data.get(SCOPE):
                         extender = VisibilityExtender(msg, VISIBILITY_TIMEOUT)
                         scope = data[SCOPE]
-                        submissionID = data[SUBMISSION_ID]
+                        submission_id = data[SUBMISSION_ID]
                         validator = MetaDataValidator(mongo_dao, model_store)
-                        status = validator.validate(submissionID, scope) 
+                        status = validator.validate(submission_id, scope)
                         if not status or status == FAILED: 
                             status = None
                         mongo_dao.set_submission_validation_status(validator.submission, None, status, None) 
                     else:
                         log.error(f'Invalid message: {data}!')
 
-                    batches_processed +=1
+                    batches_processed += 1
                     msg.delete()
+                    set_scale_in_protection(False)
```

Comment:
swipe line 61 and 62 to make sure set_scale_in_protection(False) being executed.

Reply:
updated